src/libmongoc/tests/test-mongoc-client-side-encryption.c

```diff
@@ -3530,6 +3530,20 @@ test_kms_tls_options_extra_rejected (void *unused)
    ASSERT (NULL == ce);
    mongoc_client_encryption_opts_destroy (ce_opts);
 
+   /* Test that tlsDisableOCSPEndpointCheck may be set. */
+   memset (&error, 0, sizeof (bson_error_t));
+   ce_opts = mongoc_client_encryption_opts_new ();
+   mongoc_client_encryption_opts_set_keyvault_namespace (
+      ce_opts, "keyvault", "datakeys");
+   mongoc_client_encryption_opts_set_keyvault_client (ce_opts, keyvault_client);
+   mongoc_client_encryption_opts_set_kms_providers (ce_opts, kms_providers);
+   mongoc_client_encryption_opts_set_tls_opts (
+      ce_opts, tmp_bson ("{'aws': {'tlsDisableOCSPEndpointCheck': true}}"));
```

Comment:
```suggestion
   mongoc_client_encryption_opts_set_tls_opts (
      ce_opts,
      tmp_bson ("{'aws': {'%s': true}}",
                MONGOC_URI_TLSDISABLEOCSPENDPOINTCHECK));
```
Suggest using `MONGOC_URI_TLSDISABLEOCSPENDPOINTCHECK` macro instead of hard-coded string value.